test/Content.spec.jsx

```diff
@@ -54,4 +68,58 @@ describe('The Content component', () => {
       });
     });
   });
+
+  it('should call contentWillUnmount before unmounting', () => {
```

Comment:
This doesn't seem to be testing if `willUnmount` is called?

Reply:
I didn't get you.

Reply:
Got it.